src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
 
  public void myMethod() {
   if (CoNstANT.equals("abc")) {
 
   }
+  if (CoNstANT.equals("abc")) {
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck
**Severity**: INFO
**File**: /home/bjerre/workspace/violation-comments-to-github-plugin/work/jobs/test/workspace/violations-test/src/main/java/se/bjurr/violations/lib/example/OtherClass.java L13

Must have at least one statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*